src/Microsoft.DocAsCode.DataContracts.Common/Constants.cs

```diff
@@ -48,6 +48,7 @@ public static class PropertyName
             public const string ExtensionMethods = "extensionMethods";
             public const string Overload = "overload";
             public const string Return = "return";
+            public const string SeeAlsoContent = "seealsoContent";
 
             public const string SystemKeys = "_systemKeys";
```

Comment:
not used now. can add it later

---
In reply to: [114701341](https://github.com/dotnet/docfx/pull/1625#discussion_r114701341) [](ancestors = 114701341)